pkg/dom0-ztools/rootfs/bin/eve

```diff
@@ -159,6 +160,10 @@ __EOT__
           uuid=$(cat /persist/status/uuid)
           echo "$uuid"
           ;;
+ virttype)
```

Comment:
I the rest of EVE-OS this is called "hv". I'd suggest we use the same term here.